app.py

```diff
@@ -1,48 +0,0 @@
-from flask import Flask
```

Comment:
All of our code started in this `app.py` file and was refactored into `run.py` and `models.py` within an `app` directory